packages/aragon-cli/src/lib/apm/extractContractInfoToFile.js

```diff
@@ -0,0 +1,10 @@
+const extract = require('../../helpers/solidity-extractor')
+const { writeJson } = require('fs-extra')
+
+module.exports = async (contractPath, outputPath, progressHandler) => {
```

Comment:
What's the purpose of having a `progressHandler` that gets called once at the end? I would be cleaner to remove it and just log whatever the front-end needs after this async function has resolved.